### ISSUE TITLE
how to run the tests?

### ISSUE THREAD

**Issue by fegvilela** [bibektimilsina000/FastAPI-PgStarterKit#2]
AWESOME work! 

I was wandering, though, how can I run the tests? In the "original" tiangolo version, there are a bunch of scripts that creates testing stack and run the tests on it. I'm trying to run the tests from your version with just pytest, but they're not passing 😢 

<img width="860" alt="Screenshot 2023-08-19 at 18 57 21" src="https://github.com/bibektimilsina000/FastAPI-PgStarterKit/assets/28360360/466826b8-ffe5-4f81-9985-7036ebfdc8cf">

**Comment by bibektimilsina000** [comment]
have you try running test inside dorker container?

**Comment by bibektimilsina000** [comment]
test related to celery may have fail cause i haven't work with celery right now and i dont have much understanding related to it i hope you guys helps to fix those issues also

**Comment by fegvilela** [comment]
running inside the container worked, thanks! But there are other tests failing, not just related to celery. If I can, I'll fix those ;)

**Comment by bibektimilsina000** [comment]
> running inside the container worked, thanks! But there are other tests failing, not just related to celery. If I can, I'll fix those ;)

thank you so much

**Comment by fegvilela** [comment]
Solved the issue! 
For some reason, alembic didn't create the `item` table at the 1st time. As it has a docker volume to persist the data, the migration wasn't applied again. So it still not creating the `item` table. 
I had to erase all docker volumes and then it created the `item` table. Now, all the tests (except the celery one) are ok 🎉 

![Screenshot 2023-08-23 at 21 12 48](https://github.com/bibektimilsina000/FastAPI-PgStarterKit/assets/28360360/ecb94dee-b3bd-4da0-8842-c33a17c52898)
